docs/proposals/20201026-creating-control-plane-components.md

```diff
@@ -146,7 +146,7 @@ The component controller will create the component under the in-tree mode, which
 
 Each component's controller will generate necessary certificates for the component and store them to the [secret resources](https://cluster-api.sigs.k8s.io/tasks/certs/using-custom-certificates.html) defined by CAPI. Also, The KAS controller will store the content of the kubeconfig file in a secret named `[clustername]-kubeconfig`.
```

Comment:
Missing explanation for CAPI and NCP.

Reply:
Do you mean the definition of the acronyms? This is actually @charleszheng44's first doc I just moved files around and needed to update image paths. But if that is the case all these proposals link to our internal glossary https://github.com/kubernetes-sigs/cluster-api-provider-nested/blob/master/proposals/00_capn-glossary.md which also chains to the CAPI glossary for those terms.